src/table/cells/handle-cell-focus.js

```diff
@@ -80,3 +80,31 @@ export const handleNavigateTop = ({ tableSection, focusedCellCoord, rootElement
     }
   }
 };
+
+export const getCellSrNotation = ({ focusedCellCoord, rootElement, selState, tableData }) => {
+  if (focusedCellCoord.toString().startsWith('0')) return '';
+
+  const [rowIdx, colIdx] = focusedCellCoord;
+  const rowElements = rootElement.getElementsByClassName('sn-table-row');
+  const cell = rowElements[rowIdx]?.getElementsByClassName('sn-table-cell')[colIdx];
+
+  const tColumnName = tableData.columns[colIdx].label;
```

Comment:
I don't remember, did we discuss this? Optimally the screen reader should say the column name from the markup alone. If it doesn't than maybe we can have it here.

Reply:
✅ Yes, NVDA says the column name